config/bootstrap.php

```diff
@@ -1,16 +1,4 @@
 <?php
 declare(strict_types=1);
```

Comment:
No point keeping an empty file, it can be dropped and also set the bootstrap hook property to `false` in `Plugin`. 

Also `Plugin` class should be renamed to `CakePdfPlugin` as per new convention.